URBNSwiftyConvenience/Classes/URBNBorders.swift

```diff
@@ -0,0 +1,290 @@
+//
+//  URBNBorders.swift
+//  Pods
+//
+//  Created by Lloyd Sykes on 6/30/17.
+//
+//
+
+import Foundation
+
+extension UIView {
+    public var urbn_leftBorder: URBNBorder? {
+        return leftBorder()
+    }
+    
+    public var urbn_rightBorder: URBNBorder? {
+        return rightBorder()
+    }
+    
+    public var urbn_topBorder: URBNBorder? {
+        return topBorder()
+    }
+    
+    public var urbn_bottomBorder: URBNBorder? {
+        return bottomBorder()
+    }
+    
+    public func urbn_setBorder(withColor color: UIColor, width: CGFloat) {
+        urbn_leftBorder?.width = width
+        urbn_rightBorder?.width = width
+        urbn_topBorder?.width = width
+        urbn_bottomBorder?.width = width
+        urbn_leftBorder?.color = color
+        urbn_rightBorder?.color = color
+        urbn_topBorder?.color = color
+        urbn_bottomBorder?.color = color
+    }
+    
+    public func urbn_resetBorders() {
+        urbn_borderView()?.resetBorders()
+    }
+    
+    private func urbn_borderView() -> URBNBorderView? {
+        if var borderView = objc_getAssociatedObject(self, &kURBNBorderViewKey) as? URBNBorderView {
```

Comment:
This logic is wrong. If an URBNBorderView is returned for `objc_getAssociatedObject`, then return that border view. Otherwise, create a new one with the values from below.

Currently, this will never set an associated object so this will always return nil.